react/InstallmentsList.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from 'react'
+import { defineMessages } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
+import { ProductContext } from 'vtex.product-context'
+
+import { BasicPriceProps } from './types'
+import InstallmentsRenderer, {
+  Installments,
+} from './components/InstallmentsRenderer'
+
+const CSS_HANDLES = ['installmentsListContainer'] as const
+
+function InstallmentsList(props: BasicPriceProps) {
+  const { message, markers } = props
+  const { selectedItem } = useContext(ProductContext)
+  const handles = useCssHandles(CSS_HANDLES)
+
+  const commertialOffer = selectedItem?.sellers[0]?.commertialOffer
+
+  if (
+    !commertialOffer?.Installments ||
+    commertialOffer.Installments?.length === 0
+  ) {
+    return null
+  }
+
+  const sortedInstallments = commertialOffer.Installments.sort(
+    (a: Installments, b: Installments) =>
+      a.NumberOfInstallments - b.NumberOfInstallments
+  )
+
+  return (
+    <div
+      className={`${handles.installmentsListContainer} flex flex-column items-center`}
+    >
+      {sortedInstallments.map((inst: Installments, i: number) => {
+        return (
```

Comment:
What do you think of creating two blocks?

product-installments-list and product-installments-list-item?

This way the user can place other blocks around the item


```json
{
"product-installments-list": {
  "children": [
    "flex-layout.row#installment"
  ]
},
"flex-layout.row#installment": {
  "props": {
    "blockClass": "installment"
  },
  "children": [
    "product-installment-list-item"
  ]
}
}
```